From 5d8c768a5300a71d408483bbb9535bcfe2295fcf Mon Sep 17 00:00:00 2001
From: Larry Finger <Larry.Finger@lwfinger.net>
Date: Wed, 5 Feb 2014 10:36:50 -0600
Subject: [PATCH 189/390] rtl8192du: Add BUILD_MODULE_NAME to dkms.conf

Apparently there is some confusion as to whether the symbol should
be BUILD_MODULE_NAME or BUILT_MODULE_NAME. Provide both.

Signed-off-by: Larry Finger <Larry.Finger@lwfinger.net>
---
 dkms.conf | 1 +
 1 file changed, 1 insertion(+)

--- a/drivers/staging/rtl8192du/dkms.conf
+++ b/drivers/staging/rtl8192du/dkms.conf
@@ -1,5 +1,6 @@
 MAKE="'make'"
 BUILT_MODULE_NAME=8192du
+BUILD_MODULE_NAME=8192du
 DEST_MODULE_LOCATION=/kernel/drivers/net/wireless/
 PACKAGE_NAME=8192du
 PACKAGE_VERSION=1.0
